cibuild.sh

```diff
@@ -99,26 +103,27 @@ fi
 FORCE_DOWNLOAD=true
 source "${ROOT_PATH}"/build/scripts/obtain_dotnet.sh
 
-RUNTIME_ID="$(dotnet --info | awk '/RID:/{print $2;}')"
-echo "Using Runtime Identifier: ${RUNTIME_ID}"
+if [[ "${SKIP_RESTORE}" == false ]]
+then
+    RESTORE_ARGS="-v Minimal --disable-parallel"
```

Comment:
I've moved this into `restore.sh` in post-dev15.5-contrib, so you should just be able to call that.